app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/PebbleSync.java

```diff
@@ -171,27 +171,16 @@ public int getBatteryLevel() {
     }
 
     public String slopeOrdinal(){
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method slopeOrdinal() has an NPath complexity of 256](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1312212540/issues/source?bid=2735053&fileBranchId=2735108#l173)